We do not need to check docs, formatting, coverage, typing for this project.

--- jaraco.versioning-1.1.0/setup.cfg.orig
+++ jaraco.versioning-1.1.0/setup.cfg
@@ -29,12 +29,6 @@
 [options.extras_require]
 testing = 
 	pytest >= 6
-	pytest-checkdocs >= 2.4
-	pytest-black >= 0.3.7; \
-	python_implementation != "PyPy"
-	pytest-cov
-	pytest-mypy >= 0.9.1; \
-	python_implementation != "PyPy"
 	pytest-enabler >= 1.3
 docs = 
 	sphinx >= 3.5
